### ISSUE TITLE
The module is not working on Custom System Builder

### ISSUE THREAD

**Issue by eperezsantos** [skepickle/foundryvtt-unfold-all-rolls#1]
Hi, 
The module is not working on Custom System Builder. I throw dice clicking on a label on the character sheet and the results are not shown. Instead, I select dice from the pool of dice (the lower part of the chat) and the module works fine.
Could you get it working for Custom System Builder?
Bye and many thanks

**Comment by skepickle** [comment]
Please provide information about the versions of FoundryVTT core and the CSB system being used.

**Comment by skepickle** [comment]
@eperezsantos Please provide information about the versions of FoundryVTT core and the CSB system being used. Also, any screenshots you could provide would be useful, along with details on how the CSB sheet's label is configured. Otherwise I don't have a great chance of reproducing the error on my end.

**Comment by skepickle** [comment]
I tried some stuff. As you said, using `/roll` from the chat works fine; the dice are expanded. But when using CSB system, and using a label roll formula, the system itself makes the roll look completely different from normal FoundryVTT rolls.

This module is _extremely_ simple; all it does is mess with the CSS of dice roll chat messages in order to show the dice. Also, this module is completely system-agnostic as of right now.

So, in conclusion, in order to support CSB's particular way of doing dice rolls from labels, There would have to be system-specific stuff implemented. I judge this to be out of scope for the intent of this module. Sorry, bud. Good luck with your games though!
